src/main/resources/templates/nb/vedtak/periode_vilkår.hbs

```diff
@@ -40,14 +40,14 @@ Dere har ikke fått den informasjonen dere trengte for å forstå at beløpet so
             {{#if vurderinger.unntasInnkrevingPgaLavtBeløp}}
             {{> ikke-krev-småbeløp}}
             {{else}}
-Du har fått vite om du har rett til {{{ytelsesnavnUbestemt}}} og hvor mye du har rett til. Selv hvis du har meldt fra til oss, kan vi kreve tilbake det du har fått for mye dersom du burde forstått at beløpet var feil.
+Du har fått vite om du har rett til {{{ytelsesnavnUbestemt}}} og hvor mye du har rett til. Selv hvis du har meldt fra til oss, kan vi kreve tilbake det du har fått for mye hvis du burde forstått at beløpet var feil. At du må betale tilbake, betyr ikke at du selv har skyld i feilutbetalingen.
```

Comment:
Kanskje funksjonellt, og kanskje litt språkforvirret 🇸🇪 men skriver det her:
> Selv hvis du har meldt fra til oss, kan vi kreve tilbake det du har fått for mye hvis du burde forstått at beløpet var feil

Dette er for meg en relativt vanskelig setning.
1. "mye hvis" -> "mye av, hvis"? Av er kanskje feil? Men føler jeg trenger en paus der eller noe? 
2. "hvis du burde forstått" 👀 

> betyr ikke at du selv har skyld
Jeg tolker dette som at det aldri er aldri er min skyld, og kun NAV sin skyld.. ?

Reply:
@blommish Brevtekstene er funskjonelt ja, og nok godt diskutert av de